### PR TITLE
docs: added documentation about -A option

### LINKED ISSUE #715: Falco supported events
**What to document**

After discussing with @Andreagit97 about recent changes and minor breaking changes (eg,e.g.-A` will not be applied anymore when reading from `.scap` files), we decided to create a table, so it was clear what will be the expected outcome for Falco 0.33.0. 

Ref falcosecurity/falco#2201 

The documentation (and maybe other places like the help usage of the command line) may need to be updated accordingly.

/milestone 0.33.0

cc @falcosecurity/falco-maintainers @falcosecurity/libs-maintainers

## Falco supported events

| Category | `evt.type` | Default (without -A) | With `-A` | `.scap` file |
| -------- | -------- | -------- | -------- | -------- | 
| `EC_PLUGIN` | `plugin`     | Yes | Yes | Yes |
| `EC_SYSCALL` | various     | Only syscalls in *simple_set*  | All | All, including `EF_OLD_VERSION` | 
| `EC_TRACEPOINT` | `procexit`, `switch`, `signaldeliver`, `page_fault`    | `procexit` | All |  All, including `EF_OLD_VERSION` |
| `EC_INTERNAL` | `container`, `useradded`, `userdeleted`, `groupadded`, `groupdeleted` and other internal events    | Only `container`, `useradded`, `userdeleted`, `groupadded`, `groupdeleted` | All | All, including `EF_OLD_VERSION` |

Notes:
 - `EF_OLD_VERSION` are never generated when live mode, but they may be present in `.scap` files
 - Since the new Falco version won't apply any userspace pre-filtering, `-A` is implicit when reading from `.scap`
 - `EC_PLUGIN` is not yet present in `libs`, but our plan is to insert it soon

### PR DESCRIPTION
**Pull request by therealbobo**
**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

> /kind cleanup

> /kind design

> /kind user-interface

/kind content

> /kind translation

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

> /area blog

/area documentation

> /area videos

**What this PR does / why we need it**:

This adds some documentation about the `-A` option as mentioned in the issue #715 .

**Which issue(s) this PR fixes**:

Fixes #715

**Special notes for your reviewer**:

Maybe could be a good idea to retrieve automatically the information from the code (with some grep magic?)

### CODE REVIEW

File: data/en/reference/daemon/cli_options.yaml [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+    - `umount`
+    - `unlink`
+    - `unlinkat`
+    - `unshare`
+    - `userfaultfd`
+    - `vfork`
+    - `write`
+    - `writev`
+  default: |
+    Only syscalls in *simple_set*:
```
Comment (jasondellaluce) [inline data/en/reference/daemon/cli_options.yaml:153]: Note: we are changing this to the `sinsp_state_sc_set`, plus beware of the newly-introduced `base_syscalls` and the in-review `adaptive_base_syscalls` options. My suggestion is to wait for the adaptive syscall selection feature development to be over and then update this section. Reference:
- https://github.com/falcosecurity/falco/pull/2432
- https://github.com/falcosecurity/libs/pull/826
- https://github.com/falcosecurity/falco/issues/2433

**Review by jasondellaluce — COMMENTED** [review]
Lovely! Just left a comment.

Plus, since this changes will be relled-out in Falco 0.35.0, we need to hold merging this until then to avoid having an inconsistent doc in the 0.34.X website snapshot.

/hold

File: data/en/reference/daemon/cli_options.yaml (hunk 1 above)
Comment (therealbobo) [reply data/en/reference/daemon/cli_options.yaml:153]: Makes sense! Thanks! 😄

Comment (leogr) [reply data/en/reference/daemon/cli_options.yaml:153]: cc @incertum

**Review by jasondellaluce — APPROVED** [review]
/approve

**Pull request merged**